ironmon_tracker/screens/LogOverlay.lua

```diff
@@ -93,6 +96,7 @@ LogOverlay.Windower = {
 		end
 		LogOverlay.refreshTabBar()
 		LogOverlay.refreshInnerButtons()
+		Main.SaveSettings()
```

Comment:
The `Main.SaveSettings()` call should not be here, and should be called when the user clicks on the option to change it. Check out how other checkbox options work for an example. See `TimeMachineScreen.Buttons.EnableRestorePoints`

Reply:
This was something we briefly discussed in the discord when it was being worked on, not sure of the performance impact tbf but i'm not sure how much i agree with saving (which involves making a write to the settings.ini file) every time the setting is toggled on/off. Imagine a scenario where someone maybe accidentally double-clicks, then has to click again to re-toggle. That's 3 file writes there potentially